apps/remix-ide/src/app/panels/file-panel.js

```diff
@@ -66,6 +66,15 @@ module.exports = class Filepanel extends ViewPlugin {
 
   /**
    * @param item { id: string, name: string, type?: string[], path?: string[], extension?: string[], pattern?: string[] }
+   * typically: 
+   * group 0 for file manipulations
```

Comment:
This can be improved to introduce group creations and these comments will serve as group descriptions. It will also allow plugins to create/register groups that other plugins can add items to.